portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/search/searchtuning/SearchExperiences.testcase

```diff
@@ -636,6 +636,186 @@ definition {
 		LexiconEntry.viewEntriesSpecificOrder(entryList = "Blueprint 3,Blueprint 2,Blueprint 1");
 	}
 
+	test ValidateFieldValues {
+		SearchExperiences.openSearchExperiencesAdmin();
+
+		SearchExperiences.addBlueprint(name = "Validate Field Values");
+
+		SearchExperiences.addQueryElement(element = "Text Match Over Multiple Fields");
+
+		Click(locator1 = "Icon#DELETE");
+
+		Click(locator1 = "Icon#DELETE");
+
+		Click(locator1 = "Icon#DELETE");
+
+		Button.clickSave();
+
+		AssertTextEquals(
+			locator1 = "Message#ERROR_FORM_FIELD_REQUIRED",
+			value1 = "This field is required.");
+
+		Click(
+			key_text = "Add Field",
+			locator1 = "Icon#PLUS_LAST");
+
+		Type(
+			key_fieldLabel = "Field",
+			locator1 = "SearchExperiences#BLUEPRINTS_QUERY_TEXT_FIELD",
+			value1 = "title");
+
+		Type(
+			key_fieldLabel = "Minimum Should Match",
+			locator1 = "SearchExperiences#BLUEPRINTS_QUERY_TEXT_FIELD",
+			value1 = "");
+
+		Button.clickSave();
+
+		AssertTextEquals(
+			locator1 = "Message#ERROR_FORM_FIELD_REQUIRED",
+			value1 = "This field is required.");
+
+		Type(
+			key_fieldLabel = "Minimum Should Match",
+			locator1 = "SearchExperiences#BLUEPRINTS_QUERY_TEXT_FIELD",
+			value1 = "1");
+
+		Type(
+			key_fieldLabel = "Boost",
+			locator1 = "SearchExperiences#BLUEPRINTS_QUERY_GENERIC_FIELD",
+			value1 = "");
+
+		Button.clickSave();
+
+		AssertTextEquals(
+			locator1 = "Message#ERROR_FORM_FIELD_REQUIRED",
+			value1 = "This field is required.");
+
+		Type(
+			key_fieldLabel = "Boost",
+			locator1 = "SearchExperiences#BLUEPRINTS_QUERY_GENERIC_FIELD",
+			value1 = "1");
+
+		PortletEntry.save();
+
+		Navigator.gotoNavItem(navItem = "Elements");
+
+		SearchExperiences.editElement(name = "Text Match Over Multiple Fields");
+
+		var uiConfigurationJSON = '''
+{
+	"fieldSets": [
+		{
+			"fields": [
+				{
+					"defaultValue": [
+						{
+							"field": "content",
+							"boost": "1",
+							"locale": "${context.language_id}"
+						}
+					],
+					"name": "fields",
+					"typeOptions": {
+						"boost": true
+					},
+					"label": "Fields",
+					"type": "fieldMappingList"
+				},
+				{
+					"defaultValue": "or",
+					"name": "operator",
+					"typeOptions": {
+						"options": [
+							{
+								"label": "OR",
+								"value": "or"
+							}
+						]
+					},
+					"label": "Operator",
+					"type": "select"
+				},
+				{
+					"defaultValue": "best_fields",
+					"name": "type",
+					"typeOptions": {
+						"options": [
+							{
+								"label": "Best Fields",
+								"value": "best_fields"
+							}
+						]
+					},
+					"label": "Match Type",
+					"type": "select"
+				},
+				{
+					"defaultValue": "AUTO",
+					"name": "fuzziness",
+					"typeOptions": {
+						"nullable": true,
+						"options": [
+							{
+								"label": "Auto",
+								"value": "AUTO"
+							}
+						]
+					},
+					"label": "Fuzziness",
+					"type": "select"
+				},
+				{
+					"defaultValue": "",
```

Comment:
we're setting the default value here to skip setting it when we edit the blueprint